source/style-guide/seo-guidelines.txt

```diff
@@ -0,0 +1,98 @@
+.. _seo-guidelines:
+
+=====================================
+Search Engine Optimization Guidelines
+=====================================
+
+Search Engine Optimization (SEO) considers the actual terms that users 
+enter into search engines (keywords) and employs best practices to 
+improve traffic to web pages from search engines. The following 
+guidance should help you optimize your documentation pages for search.
+
+Titles
+------
+
+Title pages and subsections according to the following SEO best 
+practices:
+
+- Use a maximum of 70 characters.
+
+- Include target keywords (the actual terms that users 
+  enter into search engines to reach your page).
+
+- Avoid excessive or irrelevant words (keyword stuffing).
+
+- Strive for unique page titles. Identical titles, even between 
+  documentation sets, might compete in search results.
+  
+To learn more, see :ref:`titles-and-headings`.
+
+Alternative Text
+----------------
+
+Screen readers read alternative text for images aloud so that users can 
+better understand an image. Specify alternative text according to the 
+following SEO best practices:
+
+- Use a maximum of 125 characters per image.
+
+- Describe the image with sufficient detail to understand what it shows.
+
+- If it makes sense, include keywords (the actual terms that users 
+  enter into search engines).
+
+To learn more, see :ref:`write-for-accessibility`.
+
+Keywords
+--------
+
+You can add the actual terms that users enter into search engines 
+(keywords) to your page to improve its SEO. Add keywords according to 
+the following best practices:
+
+- Take the most concise form of the information that the page conveys 
+  and make that the target keyword.
```

Comment:
[suggestion, non-blocking]

Would it be possible to include an example of this practice? For instance, I would find it helpful if we provided a link to some docs page and highlighted what the target keyword would be for that page.